Expensify/App
# Email address is cutoff on iOS on Login

Email address is cutoff at the bottom during log in on iOS

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/143769

### Tests (iOS)
1. Go to login page of the chat app
1. write an email that as longer letters like `g` or `p` or `j` also `T`
1. Verify that none of this letters are cut off and they are fully visible and readable.
